e2e/test/scenarios/admin/databases/add-new-database.cy.spec.js

```diff
@@ -198,6 +198,10 @@ describe("admin > database > add", () => {
 
       cy.url().should("match", /\/admin\/databases\?created=true$/);
 
+      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
```

Comment:
These tests were failing because they were looking for the `table` element that was behind the modal. With the `FocusTrap`, these elements aren't available until the modal is closed, so we need to close that modal before finding the database table again.

Reply:
Yeah makes sense - I didn't write these tests but I can make a few changes to get rid of a few of the statements